packages/browser/README.md

```diff
@@ -56,3 +56,13 @@ To enable or disable network capture dynamically, simply invoke the `enableAdvan
 ```js
 HyperDX.enableAdvancedNetworkCapture();
 ```
+
+### (Optional) React ErrorBoundary Integration
+
+To enable automatic error tracking with ErrorBoundary, simply attach the HyperDX error handler to the ErrorBoundary component.
+
+```js
+import ErrorBoundary from 'react-error-boundary';
```

Comment:
Curious if we also want to export this component for people who don't have ErrorBoundary in their app yet

Reply:
yep I was thinking about that but that'd increase sdk size a lot